requirements.txt

```diff
@@ -1,8 +1,9 @@
 cloudpickle==1.2.1
 future==0.17.1
-gym==0.13.0
+gym==0.14.0
 numpy==1.16.4
 Pillow==6.0.0
 pyglet==1.3.2
 scipy==1.3.0
 six==1.12.0
+pytest_cases
```

Comment:
Version must be specified.

In general, I would just keep the testing packages as separate